docs/basic_usage.rst

```diff
@@ -40,3 +40,27 @@ NOTE: Remember to change the secret key of your application, and insure that no
 one is able to view it. The json web tokens are signed with the secret key, so
 if someone gets that, they can create arbitrary tokens, and in essence log in
 as any user.
+
+Partially protecting routes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There may be cases where you want to use one endpoint for both protected
+and unprotected data. In these situations, you can use the jwt_optional
+decorator. This will allow the view to be called whether or not an
+authorization header is sent in the request, although if there is a header
```

Comment:
Lets change this to JWT instead of authorization header. Now that this extension supports JWTs in cookies, that isn't accurate. I probably need to go through the docs and update that in more places as well